These classes are purely virtual; the implementation is in libkdevplatformshell.

--- origsrc/kdevplatform-0.9.93/interfaces/ilaunchconfiguration.h	2009-05-20 03:20:32.000000000 -0500
+++ src/kdevplatform-0.9.93/interfaces/ilaunchconfiguration.h	2009-06-12 03:53:32.952028400 -0500
@@ -36,7 +36,7 @@ class IProject;
  * compiled application, or some script file or byte-compiled file or something else
  * Provides access to the various configurations, as well as their type and name
  */
-class KDEVPLATFORMINTERFACES_EXPORT ILaunchConfiguration
+class ILaunchConfiguration
 {
 public:
     virtual ~ILaunchConfiguration() {}
--- origsrc/kdevplatform-0.9.93/interfaces/ilaunchmode.h	2009-04-30 16:28:46.000000000 -0500
+++ src/kdevplatform-0.9.93/interfaces/ilaunchmode.h	2009-06-12 03:52:51.031387800 -0500
@@ -33,7 +33,7 @@ namespace KDevelop
  * "Execute" or "Profile"
  * @see ILauncher
  */
-class KDEVPLATFORMINTERFACES_EXPORT ILaunchMode
+class ILaunchMode
 {
 public:
     virtual ~ILaunchMode() {}
